server.py

```diff
@@ -30,6 +31,8 @@
 
 metrics_handler = MetricsHandler.instance()
 
+mapping = MappingLRUCache(args.cacheSize)
```

Comment:
can we change it to be this name, simpler
```suggestion
cache = Cache(args.cache_size)
```